packages/react-core/src/components/Panel/PanelMain.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Panel/panel';
+import { css } from '@patternfly/react-styles';
+
+export interface PanelMainProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside PanelMain */
```

Comment:
Can we change to `Content rendered inside the panel main div`. Or something like that.